### PR TITLE
Set placeholder name according to the multi-attribute login config

### PR DESCRIPTION
**Pull request by sumedhe**
### Purpose
$subject

### Related Issues
- Issue wso2/product-is#12401

### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Related PRs
- Merge after wso2/carbon-identity-framework#3707

### Security checks
- [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [x] Ran FindSecurityBugs plugin and verified report?
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

**NOTE:** The `Forgot Username` link text is not updated as it provides the "username" in recovery.

### Screenshots
When the multi-attribute login is enabled, the placeholder will be changed as follows.

image

image

image

### CODE REVIEW

File: apps/authentication-portal/src/main/webapp/basicauth.jsp [hunk 1]
```diff
@@ -118,14 +118,36 @@
     Boolean isEmailUsernameEnabled = false;
     String usernameLabel = "username";
 
+    Boolean isSelfSignUpEnabledInTenant = false;
+    Boolean isUsernameRecoveryEnabledInTenant = false;
+    Boolean isPasswordRecoveryEnabledInTenant = false;
+    Boolean isMultiAttributeLoginEnabledInTenant = false;
```
Comment (ChamathNS) [inline apps/authentication-portal/src/main/webapp/basicauth.jsp:124]: false is redundant

File: apps/authentication-portal/src/main/webapp/identifierauth.jsp [hunk 2]
```diff
@@ -39,15 +39,30 @@
     String emailUsernameEnable = application.getInitParameter("EnableEmailUserName");
     Boolean isEmailUsernameEnabled = false;
     String usernameLabel = "username";
+    Boolean isMultiAttributeLoginEnabledInTenant = false;
```
Comment (ChamathNS) [inline apps/authentication-portal/src/main/webapp/identifierauth.jsp:42]: false is redundant

**Review by ashensw — APPROVED** [review]

File: apps/authentication-portal/src/main/webapp/basicauth.jsp (hunk 1 above)
Comment (sumedhe) [reply apps/authentication-portal/src/main/webapp/basicauth.jsp]: Fixed in 1591ee73e98c6530a217973e9acc72fb277be5af

File: apps/authentication-portal/src/main/webapp/identifierauth.jsp (hunk 2 above)
Comment (sumedhe) [reply apps/authentication-portal/src/main/webapp/identifierauth.jsp]: Fixed with 1591ee73e98c6530a217973e9acc72fb277be5af

**Pull request merged**